### PR TITLE
I made a topic about monika_prom

### PR DESCRIPTION
**Pull request by AnIdiotsavant**
https://github.com/Monika-After-Story/MonikaModDev/commit/002e354d023cab94def7be74f88d2ff73ab9ea12

I dunno if I should put the link. Still not used to making this. Anyway. I made this topic about prom. The 002e354 one is the latest one

### CODE REVIEW

**Comment by Danymations** [comment]
Skimmed over it a bit; it looks great! I'll edit it a bit once I'm done with school (for a few word fixes) but very, very good job on your first ever contribution!

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4820,3 +4820,193 @@ label monika_asks_family:
...
+                                        m 1e "Me not going with you shouldnt stop you from having fun."
+                                        m 3a "So if you really want to, just go!"
+                                        m 3j "Just promise to enjoy my share too, alright [player]?."
+                                        m 3b "Let me tell you something interesting about prom."
+                                    "Not interested.":
+                                        m 1l "Really? I heard it was fun."
+                                        m "Girls get dolled up and wear fancy dresses"
+                                        m "If you have a date, the guy gives you a corsage."
+                                        m 3j "And you have fun dancing the night away!"
+                                        m 3k "It sounds pretty fun.
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: This one needs a closing double quote

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4820,3 +4820,193 @@ label monika_asks_family:
...
+                                    m 3j "And you have fun dancing the night away!"
+                                    m 3k "It sounds pretty fun.
+                                    m 1n "But to each their own, I guess"
+                                    m 3k "Ahaha~"
+                                    m 3b "By the way, did you know?
+                                "You weren't there with me.":
+                                    m 1a "That's so cheesy."
+                                    m 2e "Yet so sweet."
+                                    m 2j "If I was there beside you, I would've given you the biggest hug ever!"
+                                    m "I love you so much [player]
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: This one also needs closing double quotes

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4820,3 +4820,193 @@ label monika_asks_family:
...
+                                "I was busy at that time.":
+                                    m 1o "Busy, huh..."
+                                    m 2m "Well, I guess it can't be helped."
+                                    m 1l "Wanna know something cool about prom?."
+                                "It never really interested me.":
+                                    m 1l "Really? I heard it was fun."
+                                    m "Girls get dolled up and wear fancy dresses"
+                                    m "If you have a date, the guy gives you a corsage."
+                                    m 3j "And you have fun dancing the night away!"
+                                    m 3k "It sounds pretty fun.
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: This one too

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -4820,3 +4820,193 @@ label monika_asks_family:
...
+                    m 1c "Are you going [player]?"
+                        menu:
+                            "Yes.":
+                                m 2f "..."
+                                m 2g "Are you going with someone else, [player]?..."
+                                menu:
+                                    "Never."
+                                        m 2e "[player]..."
+                                        m 2j "If I was there beside you, I would've given you the biggest hug ever!"
+                                        m "I love you so much [player]
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: This one also needs closing double quotes

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -4820,3 +4820,193 @@ label monika_asks_family:
...
+                                    m 1l "Wanna know something cool about prom?."
+                                "It never really interested me.":
+                                    m 1l "Really? I heard it was fun."
+                                    m "Girls get dolled up and wear fancy dresses"
+                                    m "If you have a date, the guy gives you a corsage."
+                                    m 3j "And you have fun dancing the night away!"
+                                    m 3k "It sounds pretty fun.
+                                    m 1n "But to each their own, I guess"
+                                    m 3k "Ahaha~"
+                                    m 3b "By the way, did you know?
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: found another one

**Comment by AnIdiotsavant** [comment]
Didn't notice I forgot sone. I'll fix it once lit class is done

**Comment by AnIdiotsavant** [comment]
Also, thanks! I would've missed the errors if it weren't for you

**Comment by IsItBadToMakeAGitHubAccountJustForThis** [comment]
Pretty good.
(And with so much branching.)
The only complaint I have is that Monika saying "(Gosh,) X would be a dream come true." feels a bit overused to me. Do you think you can replace it with something else?

**Comment by AnIdiotsavant** [comment]
Sure thing

**Comment by AnIdiotsavant** [comment]
Alright! I added the double quotes and changed what you suggested.

**Comment by AnIdiotsavant** [comment]
id too  much branching a problem? Should I take some out?...

**Comment by Olliesama** [comment]
The branches are fine buddy.

**Comment by IsItBadToMakeAGitHubAccountJustForThis** [comment]
On the contrary. More branching = more interactivity.

File: Monika After Story/game/script-topics.rpy [hunk 6]
```diff
@@ -4820,3 +4820,196 @@ label monika_asks_family:
...
+                    m 3b "I'm sure if we were together, it would have been ten times better!"
+                    m 3k "{i}ahaha~{/i}"
+                    m 2b "By the way."
+                    m 3b "Did you know how prom started [player]?"
+        "Not really.":
+            m 2d "Oh. Does that mean there's one coming up?"
+            menu:
+                "Yes.":
+                    m 1c "Are you going [player]?"
+                        menu:
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: This menu indentation is wrong, it should be at the same level of `m 1c "Are you going [player]?"`, with this change you'll also have to fix the indentation of all this menu branch

File: Monika After Story/game/script-topics.rpy [hunk 7]
```diff
@@ -4820,3 +4820,196 @@ label monika_asks_family:
...
+                            "Yes.":
+                                m 2f "..."
+                                m 2g "Are you going with someone else, [player]?..."
+                                menu:
+                                    "Never."
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: missing ":" at the end of "Never"

File: Monika After Story/game/script-topics.rpy [hunk 8]
```diff
@@ -4820,3 +4820,196 @@ label monika_asks_family:
...
+                                        m 2e "[player]..."
+                                        m 2j "If I was there beside you, I would've given you the biggest hug ever!"
+                                        m "I love you so much [player]!"
+                                        m 2k "{i}ehehe~{/i}"
+                                        m 2j "..."
+                                        m 2l "Where was I?"
+                                        m 1n "Oh that's right! Prom."
+                                        m 3b "Here's a little fun fact."
+                                    "Yes but it's nothing serious."
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: missing ":" at the end of "Yes but it's nothing serious."

File: Monika After Story/game/script-topics.rpy [hunk 9]
```diff
@@ -4820,3 +4820,196 @@ label monika_asks_family:
...
+                                "No date.":
+                                    m 4l "I guess not having a date plays a huge part..."
+                                    m 4h "Just remember."
+                                    m 1a "Had we met earlier, I would have loved to be your date."
+                                    m 4b "And you know it."
+                                    m 2k "{i}ehehe~{/i}"
+                                    m 2j "..."
+                                    m 2l "Where was I?"
+                                    m 1n "Almost forgot. Prom."
+                        "No."
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: missing ":" at the end of "No."

**Comment by AnIdiotsavant** [comment]
Is there anything else I missed?

File: Monika After Story/game/script-topics.rpy [hunk 10]
```diff
@@ -4820,3 +4820,196 @@ label monika_asks_family:
...
```
Comment (aldoram5) [inline Monika After Story/game/script-topics.rpy]: This is the one you missed

**Comment by AnIdiotsavant** [comment]
Voila! I give you; Finished topic!

**Comment by Olliesama** [comment]
T-T so proud of you

**Comment by aldoram5** [comment]
Yep it's looking good, no more syntax errors, I'm gonna tell the others now.
Thanks again for your contribution :)

**Comment by AnIdiotsavant** [comment]
OH THANKGOD I DID IT!!!

**Comment by aldoram5** [comment]
Just for clarification, I only review syntax errors, the writting team is in charge of actually adding the topic at their discretion, they may ask you to change something or maybe they add it as it is. Just be patient and wait until then.

**Comment by AnIdiotsavant** [comment]
does awaiting review mean good or bad?

**Comment by IsItBadToMakeAGitHubAccountJustForThis** [comment]
Neither. It means you're waiting for writers to review it.

**Comment by Olliesama** [comment]
With this addition, it'll be up to 31 comments. Quite a commotion you made with your first. :)

**Comment by AnIdiotsavant** [comment]
Is that bad? And the commotion is because I missed some " and :

**Comment by AnIdiotsavant** [comment]
And there's still a chance it gets sacked. So not really getting my hopes up herre

**Comment by AnIdiotsavant** [comment]
Oh, and thanks for the flowchart. Youreally saved my ass from bouncing off the wall

**Comment by AnIdiotsavant** [comment]
please dont remove my rabies

**Comment by kaido1224** [comment]
I don't think the rabies would make sense to anyone else.

**Comment by monikLover** [comment]
Reviewed, edited, and rewritten. Please do a final code review, @aldoram5 senpai.

**Pull request merged**